pootle/apps/pootle_language/views.py

```diff
@@ -40,9 +59,13 @@ def permission_context(self):
     def url_kwargs(self):
         return {"language_code": self.object.code}
 
+    @check_and_redirect_to_canonical_lang_on_404
```

Comment:
We can get rid of the decorator and incorporate the logic in `get_object` — there's no reuse, so no need to _complicate_ `dispatch` with a decorator.